plugins/modules/system/sefcontext.py

```diff
@@ -22,6 +22,13 @@
     type: str
     required: yes
     aliases: [ path ]
+  equiv:
+    description:
+    - set target selinux equivalence to this path.
```

Comment:
Please put proper sentences in each paragraph.
```suggestion
    - Set target selinux equivalence to this path.
```

Reply:
Unresolving, as the issue remains.